zcash_client_sqlite/src/lib.rs

```diff
@@ -523,13 +526,41 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> WalletRead for W
         account: AccountId,
         max_height: BlockHeight,
     ) -> Result<HashMap<TransparentAddress, NonNegativeAmount>, Self::Error> {
-        wallet::transparent::get_transparent_address_balances(
+        wallet::transparent::get_transparent_balances(
```

Comment:
This reverts a change in #1402; I was not aware that it was intentional.

Reply:
If there is no strong opinion about this, I'd prefer to leave it with the trait and concrete names consistent.